bin/update/03-run.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -e
+
+RELEASE=$1
+UMBREL_ROOT=$2
+UMBREL_USER=$3
+
+echo
+echo "======================================="
+echo "============= OTA UPDATE =============="
+echo "======================================="
+echo "=========== Stage: Success ============"
+echo "======================================="
+echo
+
+# Delete previous (unused) images
+echo "Deleting previous images"
+cat <<EOF > $UMBREL_ROOT/statuses/update-status.json
+{"state": "installing", "progress": 90, "description": "Deleting previous images"}
+EOF
+# docker image prune --all --force
```

Comment:
Yeah although I agree that it can be a bit dangerous, but I think we can still go forward with it because:
- If an advanced user has set up a custom Umbrel installation, they probably won't be using OTA update functionality out-of-the-box (they'd need to install `fswatch`, run `karen` as root, etc).
- If they're then manually installing updates via Git, this wouldn't be a problem for them. 
- If they indeed install `fswatch` and run `karen`, we can assume that their technical competency is adequate enough to audit/remove the pruning logic from the update module (provided we warn about the pruning in our docs, which we should).
- Also, most importantly it keeps us focused for now and engineer a better pruning logic later. 99% of the people would be running Umbrel via Umbrel OS, so we can go ahead and assume that all of their Docker images and volumes are Umbrel-related.

Reply:
I was thinking Umbrel users who run Umbrel OS but SSH in and run their own containers.

But yeah I agree, a simple warning stating an update nukes docker containers is sufficient. People who need to can handle it themselves.